index.html

```diff
@@ -16,7 +16,7 @@
                
                <div class="header-navbar-image">
                    <div class="flex-container-img">
-                    <img src="assets\images\toolbar-mobile.png">
+                    <img src="assets/images/toolbar-mobile.png" alt="image icon logo">
```

Comment:
- [ ] Kindly remove the image logo from your nav tag, only the hamburger section should have a nav tag, also make sure your logo has a link `<a>` tag.

Reply:
- [ ] Kindly make sure at this stage you only have a hamburger menu in your nav items, as the project is beginning with a mobile version, the other sub-menus will be implemented later.